### ISSUE TITLE
don't dependent on VS Code internal modules

### ISSUE THREAD

**Issue by jrieken** [pflannery/vscode-versionlens#7]
I am using this extension and with latest (alpha) VS Code I see many errors because [this](https://github.com/pflannery/vscode-versionlens/blob/537660e8af791e972e81e0ac0d020208c637f08a/src/bootstrap.js#L14) doesn't exist anymore. You should **not** have dependencies like that. Luckily, the code question is now a node module and @aeschli can tell how and where to get it.

```
//Applications/Visual Studio Code - Alpha.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:73 Activating extension `pflannery.vscode-versionlens` failed: Cannot find module '/Applications/Visual Studio Code - Alpha.app/Contents/Resources/app/extensions/json/server/out/utils/httpRequest'.t.$localShowMessage @ //Applications/Visual Studio Code - Alpha.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:73
index.html:28 [uncaught exception]: TypeError: Cannot read property 'startLineNumber' of nullonError @ index.html:28
```

**Comment by aeschli** [comment]
Check out node nodules https://www.npmjs.com/package/jsonc-parser and https://www.npmjs.com/package/request-light

**Comment by pflannery** [comment]
awesome. will check it out and update. thanks

**Comment by pflannery** [comment]
This is fixed in 0.4.1

I couldn't use the jsonc-parser as that doesn't return a json graph. 
https://www.npmjs.com/package/request-light worked

Funny thing is I had already extracted the json parser from vscode when I created this project. I held off from publishing it because I wanted to create the [vscode-contrib](https://github.com/vscode-contrib) repo first. 
I will also be transferring this project to this new repo soon.
